__mocks__/openmrs-esm-styleguide.mock.tsx

```diff
@@ -0,0 +1,5 @@
+export const showToast = jest.fn();
+
+beforeEach(() => {
+  showToast.mockReset();
+});
```

Comment:
This kind of thing doesn't belong in a mock. Let tests do this themselves.

Reply:
Resolved